app-text/booksorg/booksorg-0.3.recipe

```diff
@@ -0,0 +1,78 @@
+SUMMARY="A Simple Books Organizer"
+DESCRIPTION="Application to Organize with ease your pdf book"
+COPYRIGHT="2014-2018 Abou Zakaria"
+LICENSE="GNU GPL v3"
+REVISION="1"
+HOMEPAGE="http://booksorg.sourceforge.net"
+SOURCE_URI="https://downloads.sourceforge.net/booksorg/booksorg_03.tar.gz"
+SOURCE_DIR="booksorg_03"
```

Comment:
Thanks @mazbrili for your PR!
Could you please apply the following (neutral or cosmetic) changes? Thanks!
```diff
-HOMEPAGE="http://booksorg.sourceforge.net"
+HOMEPAGE="http://booksorg.sourceforge.net/"
-SOURCE_URI="https://downloads.sourceforge.net/booksorg/booksorg_03.tar.gz"
+SOURCE_URI="https://downloads.sourceforge.net/booksorg/booksorg_${portVersion/./}.tar.gz"
-SOURCE_DIR="booksorg_03"
+SOURCE_DIR="booksorg_${portVersion/./}"
 CHECKSUM_SHA256="dcdcce89090d1539c020412d294e2f6c0ffc84a0f4f400132f57e1c1be29d234"
-PATCHES="booksorg-0.3.patchset"
+PATCHES="booksorg-$portVersion.patchset"
```
(Everything else looks OK and I'll give it a try asap.)

Reply:
@fbrosson ..its done, thanks